PublicationsRouter/api/v3/Send.md

```diff
@@ -105,28 +105,33 @@ If you are sending only the notification JSON, the request must take the form:
 
 ### 2. Validate Metadata + Package request
 
-If you are sending binary content as well as the metadata, the request must take the form:
+If you are sending binary content as well as the metadata, the request should be formed using [RFC 2387](https://www.ietf.org/rfc/rfc2387.txt):
 
     POST /validate?api_key=<api_key>
     Header:
-        Content-Type: multipart/form-data; boundary=FulltextBoundary
+        Content-Type: multipart/related; boundary=FullTextBoundary
 
     Body:
-        --FulltextBoundary
-
-        Content-Disposition: form-data; name="metadata"
+        --FullTextBoundary
+        Content-Disposition: name="metadata"
         Content-Type: application/json
 
         {Incoming Notification JSON}
 
-        --FulltextBoundary
+        --FullTextBoundary
 
-        Content-Disposition: form-data; name="content"
+        Content-Disposition: name="content"
         Content-Type: application/zip
 
-        Binary Package
+        {Binary Package}
 
-        --FulltextBoundary--
+        --FullTextBoundary--
+
+This is quite simple to do in curl by using the -F flag:
+
+```bash
+curl -H 'Content-Type: multipart/related' -F 'metadata=@metadata.json;type=application/json;filename="metadata.json"' -F 'content=@myzip.zip;type=application/zip;filename="content.zip"' https://pubrouter.jisc.ac.uk/api/v3/validate?api_key=<my_api_key>
+```
```

Comment:
-1- I think that Lines 130 to 134 should go after the following sentence, 

-2- And, should be preceded by a sub-heading `#### Testing using curl

-3- Can you make the script display on several lines (so it is all visible without scrolling)?